.travis.yml

```diff
@@ -25,14 +25,14 @@ install:
 before_script:
   # static code analysis
   #   pyflakes: mainly warnings, unused code, etc.
-  - python -m pyflakes .
+  - python -m pyflakes opmd_validator
```

Comment:
I think these need an update as well for the package name

Reply:
Argh, it slipped through. Fixed.